sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample8_KeyRotation.md

```diff
@@ -33,7 +33,7 @@ Debug.WriteLine($"{cloudRsaKey.KeyType} key is returned with name {cloudRsaKey.N
 ## Set key rotation policy
 
 To set a key rotation policy, you create a new `KeyRotationPolicy` with 1 or more lifetime actions and an expiration date
-for the key that will be created when rotated.
+for the key that will be created when rotated. Note, the date format used by ExpiresIn and TimeBeforeExpiry is ISO 8601.
```

Comment:
Thanks for the suggestion, but I have a few small changes since these are ISO 8601 *durations*, which is a separate section of ISO 8601 (or RFC 3339).

To note, the API documentation does mention this, but it's great to call out for the samples as well. Thanks for catching this!
```suggestion
for the key that will be created when rotated. The date formats for `ExpiresIn`, `TimeAfterCreate `, and `TimeBeforeExpiry ` are all ISO 8601 durations, such as "P90D" meaning "90 days".
```